static/templates/settings/deactivation_stream_modal.hbs

```diff
@@ -8,6 +8,6 @@
     </div>
     <div class="modal-footer">
         <button class="button rounded close-modal-btn" data-dismiss="modal">{{t "Cancel" }}</button>
-        <button class="button rounded btn-danger" id="do_deactivate_stream_button" data-stream-id="{{stream_id}}">{{t "Yes, archive this stream" }}</button>
+        <button class="button rounded btn-danger" id="do_deactivate_stream_button" data-stream-id="{{stream_id}}">{{t "Confirm" }}</button>
```

Comment:
<img width="425" alt="archieve_stream_confirm_modal" src="https://user-images.githubusercontent.com/59444243/113510032-39015400-9576-11eb-9541-44a3c68b4338.PNG">

Reply:
To make it more consistent with the other confirmations, let's have it say:

"Archiving stream **announce** will immediately unsubscribe everyone. This action is permanent and cannot be undone.

Are you sure you want to archive this stream?

Reply:
@nikhilmaske-2001 are you doing a PR for these suggested edits?

Reply:
Oh I see, https://github.com/zulip/zulip/pull/18073.  Great.